$OpenBSD$
Allow libvpx to be detected.

The svc_context.h is used by WebRTC code, which we disable anyway.

Index: config.tests/libvpx/libvpx.cpp
---  config.tests/libvpx/libvpx.cpp.orig
+++  config.tests/libvpx/libvpx.cpp
@@ -26,7 +26,7 @@
 **
 ****************************************************************************/
 
-#include <vpx/svc_context.h>
+// required for WebRTC #include <vpx/svc_context.h>
 #include <vpx/vpx_frame_buffer.h>
 
 int main(int, char **)
